packages/components/src/progressive-disclosure-panel/stories/index.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * External dependencies
+ */
+import styled from '@emotion/styled';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import ProgressiveDisclosurePanel from '../';
+import Panel from '../../panel';
+import UnitControl from '../../unit-control';
+
+export default {
+	title: 'Components/ProgressiveDisclosurePanel',
+	component: ProgressiveDisclosurePanel,
+};
+
+export const _default = () => {
+	const [ height, setHeight ] = useState();
+	const [ width, setWidth ] = useState();
+
+	const resetAll = () => {
+		setHeight( undefined );
+		setWidth( undefined );
+	};
+
+	return (
+		<PanelWrapperView>
+			<Panel>
+				<ProgressiveDisclosurePanel
+					label="Display options"
+					title="Progressive Disclosure Panel"
+					resetAll={ resetAll }
+				>
+					<PlaceholderControl
```

Comment:
We need to update the story in accordance with the latest changes.

Reply:
✅ Done.